graphql_schema.json

```diff
@@ -9398,7 +9398,7 @@
                     "name": null,
                     "ofType": {
                       "kind": "SCALAR",
-                      "name": "String",
+                      "name": "ID",
```

Comment:
Not sure if this is right 🤔

Reply:
Yeah, why did this change? 😕

Reply:
This just changed after running GraphQL schema generation. This now aligns with the type of `default_cohort_id` in `update_course` mutator. Guess schema was not generated after this change in server.